Rakefile

```diff
@@ -159,9 +169,32 @@ def create_post_file!(filename, response, date, host)
 end
 
 def parse_title(title)
-  "\"#{title.gsub(/"/,  '')}\""
+  first, *rest = title.split # e.g. if title = "a b c", first = "a", rest = ["b", "c"]
+  first.downcase! # mutate first word to lowercase in place
+  rest.shift if UNDESIRED_PR_TITLE_WORDS.include?(rest[0]) # rm 1st word if indesired
```

Comment:
s/indesired/undesired

Reply:
Thanks! That's French slipping into my English :D